warehouse/accounts/views.py

```diff
@@ -1089,7 +1089,7 @@ def _error(message):
     request.db.add(
         JournalEntry(
             name=project.name,
-            action=f"accepted {desired_role} {user.username}",
+            action=f"add {desired_role} {user.username}",
```

Comment:
Let's leave legacy JournalEntry as is.
```suggestion
            action=f"accepted {desired_role} {user.username}",
```